var/spack/repos/builtin/packages/clingo-bootstrap/package.py

```diff
@@ -67,3 +69,10 @@ def setup_build_environment(self, env):
 
         env.set('CXXFLAGS', opts)
         env.set('LDFLAGS', opts)
+
+    @property
+    def site_packages_dir(self):
+        libraries = llnl.util.filesystem.FileList(llnl.util.filesystem.find(
```

Comment:
Should we use `LibraryList` instead of `FileList`? Also, many libraries on macOS use `.so` instead of `.dylib`. We should really have `find_libraries` search for both on macOS.

Is the directory always the same regardless of OS and whether we use internal or external Python? If so we can just hard-code it.

Reply:
> Should we use LibraryList instead of FileList? Also, many libraries on macOS use .so instead of .dylib. We should really have find_libraries search for both on macOS

I agree with this, but I would separate larger refactoring from small patches. Larger refactoring needs a closer scrutiny and are less likely to have a fast track. If they're coupled with a bugfix, the bug will stay longer.

Reply:
Outdated too